posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -443,6 +444,21 @@ def map_scroll_property(property: Union[EventPropertyFilter, PersonPropertyFilte
         ]
         return property_to_expr(properties, team=self.team, scope="event")
 
+    def _event_properties_for_bounce_rate(self) -> ast.Expr:
+        # Exclude pathname filters for bounce rate calculation
+        #
+        # This provides consistent bounce rates when filtering by multiple pathnames.
+        # Without this, pathname filters would affect which sessions are considered for the
+        # bounce rates calculations but since we group them by entry_pathname, the results could be misleading
+        # as the events would be filtered by a IN(pathname) and the bounce shown would be for the first pathname
+        # which users are not necessarily expecting to see.
+        properties = [
+            p
+            for p in self.query.properties + self._test_account_filters
+            if not (get_property_type(p) == "event" and get_property_key(p) == "$pathname")
+        ]
+        return property_to_expr(properties, team=self.team, scope="event")
```

Comment:
This is not clear to me. Can you give me an example of a filter that would give wrong results and why?

Reply:
Sure, the Zendesk ticket has more info, but this one exemplifies: it: https://us.posthog.com/project/2/web?date_from=-30d&date_to=&interval=day&device_tab=DEVICE_TYPE&source_tab=CHANNEL&graphs_tab=UNIQUE_USERS&path_tab=PATH&geography_tab=HEATMAP&path_cleaning=true&filter_test_accounts=true&compare_filter=%7B%22compare%22%3Atrue%7D&domain=all&tile_visualizations=%7B%7D&filters=%5B%7B%22key%22%3A%22%24pathname%22%2C%22value%22%3A%5B%22%2Fweb-analytics%22%2C%22%2F%22%5D%2C%22operator%22%3A%22exact%22%2C%22type%22%3A%22event%22%7D%5D

Remove the "/" pathname and see that the conversion rate will change entirely because it won't consider the sessions with "/" as a pathname anymore. Which is "sqly" correct, but as the other values (views/visitors) don't change, it is very weird.

Reply:
@rafaeelaudibert this one brain melted us to grasp it, so feel free to ask for more explanation as you need!